consensus/blocks.go

```diff
@@ -163,6 +133,8 @@ func (s *State) AcceptBlock(b Block) (rewoundBlocks []Block, appliedBlocks []Blo
 	_, exists := s.badBlocks[b.ID()]
 	if exists {
 		err = errors.New("block is known to be invalid")
+		errStr := fmt.Sprintf("%v", b.ID())
+		panic(errStr)
```

Comment:
not much point to the `err` if you're going to panic...I'm guessing the panic won't stick around forever.